cmd/config/config.go

```diff
@@ -448,6 +448,11 @@ func (c Config) Merge() Config {
 					ckvs.Set(kv.Key, kv.Value)
 				}
 			}
+			if _, ok := cp[subSys]; !ok {
+				// A config subsystem was removed or server was downgraded.
```

Comment:
We don't really support downgrades, but a defensive check is okay.

Reply:
Indeed.